tests/fsharp/core/reflect/test2.fs

```diff
@@ -300,6 +300,25 @@ module TEst =
   let _ = printany printany (* =) *)
 
 
+module DynamicCall = 
+    open System
+    open System.Reflection
+
+    module Test =
+        type Marker = class end
+
+        let inline call (a : ^a) =
+            (^a : (member Stuff : Unit -> Unit) a)
+
+
+    let genericCallMethod = typeof<Test.Marker>.DeclaringType.GetMethod "call"
+    let callMethod = genericCallMethod.MakeGenericMethod [| typeof<Object> |]
+
+    try 
+       callMethod.Invoke (null, [| Object () |]) |> ignore
+       failwith "expected an exception"
+    with :? TargetInvocationException as ex ->
+        test "wcnr0vj" (ex.InnerException.Message.Contains("Dynamic invocation"))
```

Comment:
@dsyme, If a developer runs the tests with a german or Italian local to pick two at random, will the text still contain the words "Dynamic Invocation"   ?